pandas/tests/groupby/aggregate/test_cython.py

```diff
@@ -312,3 +314,69 @@ def test_cython_agg_nullable_int(op_name):
         # so for now just checking the values by casting to float
         result = result.astype("float64")
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("with_na", [True, False])
+@pytest.mark.parametrize(
+    "op_name, action",
+    [
+        # ("count", "always_int"),
+        ("sum", "large_int"),
+        # ("std", "always_float"),
+        ("var", "always_float"),
+        # ("sem", "always_float"),
```

Comment:
BTW, the reason those are commented out, it because of `std` (and thus `sem`) is taking a different code path (not fully sure why, though, it is taking a different path compard to eg `var`)

And `count` is not actually a cython function, so also not yet covered by this PR (so maybe we should move those tests out of the ``test_cython.py`` file)